package.json

```diff
@@ -65,10 +65,12 @@
     "nodemailer": "~1.11.0",
     "nodemailer-smtp-transport": "~0.1.13",
     "request": "2.88.0",
+    "smpp": "0.3.1",
+    "swagger-jsdoc": "~3.4.0",
```

Comment:
documentation/development.md documentation should be modified to include a section about swagger (similar to what it has been done i n https://github.com/telefonicaid/iotagent-ul/pull/412)

Reply:
readme.md updated in 9c80eb9